scripts/reindex.py

```diff
@@ -199,6 +229,10 @@ def conda_data_wrapper(args):
         # write summary map of pypi package name to conda channel/name
         (PUBLIC / "conda.json").write_text(json.dumps(CONDA_INDEX, indent=2))
 
+        # update the main index (summary) with the conda versions
+        for pkg in PYPI_INDEX:
+            pkg["conda_versions"] = data.get(pkg["name"], {}).get("versions", [])
```

Comment:
Could we add a sorter helper function to make sure the versions are provided in descending order 

Something like:

```python
from packaging import version

vers = ["1.7.0", "1.7.0rc2", "1.7.0rc1", "1.7.1", "1.11.0"]


def sort_versions(versions, reverse=True):
     return list(sorted(versions, key=version.Version, reverse=reverse))
     

sort_versions(versions)
```